tests/integration/tx/context_test.go

```diff
@@ -0,0 +1,74 @@
+package tx
+
+import (
+	"testing"
+
+	"cosmossdk.io/depinject"
+	"cosmossdk.io/log"
+	"cosmossdk.io/x/tx/signing"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/proto"
+
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/tests/integration/tx/internal/pulsar/testpb"
+	"github.com/cosmos/cosmos-sdk/testutil/configurator"
+	simtestutil "github.com/cosmos/cosmos-sdk/testutil/sims"
+)
+
+func ProvideCustomGetSigners() signing.CustomGetSigner {
```

Comment:
nice, it will be useful if this could be displayed in the docs ref: https://github.com/cosmos/cosmos-sdk/issues/16294